### ISSUE TITLE
JP24

### ISSUE THREAD

**Issue by mattgarrish** [daisy/epub-accessibility-iso#24]
Although we agree that DRM often prohibits users from using accessible reading systems, we are not sure if it is a good idea to have this normative clause. As an example, consider DRM-unaware reading systems that fail to support screen readers. They are not accessible, but they are not prohibited by this clause.

Proposed Change: We propose to create a separate document for requirements on accessible reading systems.

(Clause 10)

**Comment by mattgarrish** [comment]
This section only says that the author must not enable DRM features that knowingly take away access to the content. It's not saying anything about what the reading system itself has to do or provide.

**Comment by mattgarrish** [comment]
partially accepted

Actions: Same as JP 2 and JP 3.

**Comment by murata2makoto** [comment]
In ISO, the term "required", "recommended", "shall", "should", etc. imply a normative requirement.  Please avoid them in informative sections.
